breadcord/bot.py

```diff
@@ -57,7 +59,7 @@ def discover_modules(self) -> None:
 
     def reload_settings(self) -> None:
         settings = config.Settings()
-        settings.set_schema('breadcord/settings_schema.toml')
+        settings.set_schema('settings_schema.toml')
```

Comment:
This is an incorrect change, and `breadcord/` should be prefixed if the bot is run with the correct command (`python -m breadcord`).